src/main/scala/iog/psg/cardano/CardanoApi.scala

```diff
@@ -76,166 +83,164 @@ trait CardanoApi {
+    * @param fromWalletId wallet's id
+    * @param passphrase The wallet's master passphrase. [ 0 .. 255 ] characters
+    * @param payments A list of target outputs ( address, amount )
+    * @param withdrawal Optional, when provided, instruments the server to automatically withdraw rewards from the source
+    *                   wallet when they are deemed sufficient (i.e. they contribute to the balance for at least as much
+    *                   as they cost).
+    * @param metadata   Extra application data attached to the transaction.
+    * @return create transaction request
+    */
+  def createTransaction(
+    fromWalletId: String,
+    passphrase: String,
+    payments: Payments,
+    metadata: Option[TxMetadataIn],
+    withdrawal: Option[String]
+  ): Future[CardanoApiRequest[CreateTransactionResponse]]
 
   /**
-   * Estimate fee for the transaction. The estimate is made by assembling multiple transactions and analyzing the
-   * distribution of their fees. The estimated_max is the highest fee observed, and the estimated_min is the fee which
-   * is lower than at least 90% of the fees observed.
-   * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransactionFee #estimateFee]]
-   *
-   * @param fromWalletId wallet's id
-   * @param payments A list of target outputs ( address, amount )
-   * @param withdrawal Optional, when provided, instruments the server to automatically withdraw rewards from the source
-   *                   wallet when they are deemed sufficient (i.e. they contribute to the balance for at least as much
-   *                   as they cost).
-   * @param metadataIn Extra application data attached to the transaction.
-   * @return estimate fee request
-   */
-  def estimateFee(fromWalletId: String,
-                  payments: Payments,
-                  withdrawal: Option[String],
-                  metadataIn: Option[TxMetadataIn] = None
-                 ): Future[CardanoApiRequest[EstimateFeeResponse]]
+    * Estimate fee for the transaction. The estimate is made by assembling multiple transactions and analyzing the
+    * distribution of their fees. The estimated_max is the highest fee observed, and the estimated_min is the fee which
+    * is lower than at least 90% of the fees observed.
+    * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransactionFee #estimateFee]]
+    *
+    * @param fromWalletId wallet's id
+    * @param payments A list of target outputs ( address, amount )
+    * @param withdrawal Optional, when provided, instruments the server to automatically withdraw rewards from the source
+    *                   wallet when they are deemed sufficient (i.e. they contribute to the balance for at least as much
+    *                   as they cost).
+    * @param metadataIn Extra application data attached to the transaction.
+    * @return estimate fee request
+    */
+  def estimateFee(
+    fromWalletId: String,
```

Comment:
Can we line them up as in the original? Against the open bracket of the method?

Reply:
ok, pushed commit with:
`align.openParenDefnSite = true`